website/api/blockchain-integration/contracts/verifier.md

```diff
@@ -54,20 +54,24 @@ We are working on process to provide opt-in upgradeability for the verifier cont
 RISC Zero provides an already deployed verifier contract in the Sepolia network for your convenience.
 You can choose to use this contract or deploy your own.
 
-| Contract                      | Network   | Address                                                        |
-| ----------------------------- | --------- | -------------------------------------------------------------- |
-| [RiscZeroGroth16Verifier.sol] | [Sepolia] | [0xe57255C10291771A2E14f7eb257555AE82D78302][sepolia-verifier] |
+| Contract                      | Version                      | Network   | Address                                                                |
+| ----------------------------- | ---------------------------- | --------- | ---------------------------------------------------------------------- |
+| [RiscZeroGroth16Verifier.sol] | [v0.9.0][contracts-v0.9.0]   | [Sepolia] | [0xe57255C10291771A2E14f7eb257555AE82D78302][sepolia-verifier-v0.9.0]  |
+| [RiscZeroGroth16Verifier.sol] | [v0.10.0][contracts-v0.10.0] | [Sepolia] | [0x4e9559f1fdf82dd1babdd42b4550ffdb1a1c59de][sepolia-verifier-v0.10.0] |
 
 [RiscZeroGroth16Verifier.sol]: https://github.com/risc0/risc0-ethereum/blob/release-0.8/contracts/src/groth16/RiscZeroGroth16Verifier.sol
 [IRiscZeroVerifier.sol]: https://github.com/risc0/risc0-ethereum/blob/main/contracts/src/IRiscZeroVerifier.sol
 [EvenNumber.sol]: https://github.com/risc0/bonsai-foundry-template/blob/main/contracts/EvenNumber.sol
 [article-groth16]: https://www.risczero.com/news/on-chain-verification
 [article-proof-composition]: https://www.risczero.com/news/proof-composition
 [Sepolia]: https://ethereum.org/nb/developers/docs/networks#sepolia
-[sepolia-verifier]: https://sepolia.etherscan.io/address/0xe57255C10291771A2E14f7eb257555AE82D78302#code
+[sepolia-verifier-v0.9.0]: https://sepolia.etherscan.io/address/0xe57255C10291771A2E14f7eb257555AE82D78302#code
+[sepolia-verifier-v0.10.0]: https://sepolia.etherscan.io/address/0x4e9559f1fdf82dd1babdd42b4550ffdb1a1c59de#code
 [term-journal]: /terminology#journal
 [term-receipt]: /terminology#receipt
 [term-verify]: /terminology#verify
 [term-image-id]: /terminology#image-id
 [term-zkvm-program]: /terminology#zkvm-program
 [foundry-template]: https://github.com/risc0/bonsai-foundry-template
+[contracts-v0.9.0]: https://github.com/risc0/risc0-ethereum/releases/tag/v0.9.0
```

Comment:
Are we better off pointing to the release notes or to the [contract from that release](https://github.com/risc0/risc0-ethereum/blob/v0.9.0/contracts/src/IRiscZeroVerifier.sol)?

Reply:
My intention is to have the "contract" column link to the source code, and the "version" column link to the release.
I've updated the link tah in f2d4a006571565d30422ab77a0cdec3a21c85914